### PR TITLE
Move testgrid config upload to prow postsubmit

### PR DESCRIPTION
**Pull request by krzyzacy**
#1246 @spxtr is this enough?

Don't need to waste resource run it every 10min anymore, yay.

### CODE REVIEW

**Comment by spxtr** [comment]
You should run it as a kubernetes pod rather than a Jenkins job.

**Comment by krzyzacy** [comment]
updated, made a gcloud image

**Comment by rmmh** [comment]
@k8s-bot test this 

(testing that a deploy worked, ignore)

**Comment by rmmh** [comment]
@k8s-bot test this

(testing that a deploy worked, ignore)

**Comment by krzyzacy** [comment]
how did the test go?

**Comment by rmmh** [comment]
It worked! :)

**Comment by krzyzacy** [comment]
do I still need to run `make update-jobs` afterwards, as the prow doc indicates?

**Comment by spxtr** [comment]
You will need to, yes.

**Comment by krzyzacy** [comment]
@spxtr PTAL then?

**Comment by spxtr** [comment]
Looks almost right, but you need some equivalent of `images/pull-kubernetes/bazel/runner`.

File: images/gcloud/runner [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+set -o errexit
+set -o nounset
+set -o pipefail
+
+git clone https://github.com/kubernetes/test-infra
+./test-infra/jenkins/bootstrap.py \
+    --repo=k8s.io/${REPO_NAME} \
+    --job=${JOB_NAME} \
+    --service-account=${GOOGLE_APPLICATION_CREDENTIALS} \
+    "$@"
```
Comment (spxtr) [inline images/gcloud/runner]: Also set `--branch=${PULL_REFS}` and `--upload=gs://kubernetes-jenkins/logs`.

Comment (krzyzacy) [inline images/gcloud/runner]: why do I care about --branch

Comment (spxtr) [inline images/gcloud/runner]: Otherwise how does bootstrap know what to build?

File: images/gcloud/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+ENV GCLOUD_VERSION 138.0.0
+RUN wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-$GCLOUD_VERSION-linux-x86_64.tar.gz && \
+    tar xf google-cloud-sdk-$GCLOUD_VERSION-linux-x86_64.tar.gz && \
+    rm google-cloud-sdk-$GCLOUD_VERSION-linux-x86_64.tar.gz && \
+    ./google-cloud-sdk/install.sh
+ENV PATH "/google-cloud-sdk/bin:${PATH}"
+
+WORKDIR /workspace
+ENTRYPOINT ["/bin/bash"]
```
Comment (spxtr) [inline images/gcloud/Dockerfile]: Also `ADD runner /runner` and `ENTRYPOINT ["/runner"]`

Comment (krzyzacy) [inline images/gcloud/Dockerfile]: forgot to push :-p done

File: images/gcloud/runner (hunk 1 above)
Comment (krzyzacy) [inline images/gcloud/runner]: added to `postsubmit.yaml`

**Pull request merged**